addons/website_forum/controllers/website_forum.py

```diff
@@ -140,9 +152,8 @@ def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=Non
                                       step=self._post_per_page, scope=self._post_per_page,
                                       url_args=url_args))
 
-        values = self._prepare_user_values(forum=forum, searches=post, header={'ask_hide': not forum.active})
+        values = self._prepare_user_values(forum=forum, searches=post, header={'ask_hide': bool(forum) and not forum.active})
```

Comment:
Did you find code using `ask_hide`?

Reply:
I see that 'ask_hide' is used in Odoo 14, in the header template:

`<div t-if="not header.get('ask_hide')" t-attf-class="btn-group btn-block mb16 #{(user.karma &lt; forum.karma_ask) and not is_public_user and 'karma_required' or ''}" t-attf-data-karma="#{forum.karma_ask}">`

I can't find any trace of it in the current version of Odoo. This is indeed dead code. I remove it